package.json

```diff
@@ -11,7 +11,8 @@
     "test": "tests"
   },
   "scripts": {
-    "test": "grunt test-dev"
+    "ci": "karma start karma.conf.js --single-run",
+    "test": "karma start karma.conf.js"
```

Comment:
`npm test-dev` is what I use for non-single-runs in a lot of my projects. I'm also super open to any other names.